src/openassetio-python/CMakeLists.txt

```diff
@@ -5,3 +5,9 @@
 add_subdirectory(bridge)
 # Python C extension module making C++ types usable in Python.
 add_subdirectory(module)
+
+# Install OpenAssetIO pure Python into the venv.
+install(
```

Comment:
[question] CMake noob Q - how do we then add this as a dependency later? So when we run the tests, we know this is definitely done?

Reply:
`install`ed things are added to the built-in `install` build target, which we wrap as `openassetio.internal.install`.  So when CMake installs the binary libraries, it also installs these files.